src/components/share-drawer/utils.ts

```diff
@@ -0,0 +1,6 @@
+export const getTwitterLink = (url: string | null, text: string) => {
```

Comment:
Can you update `TwitterShare.tsx` to use this function

Reply:
yeah definitely, regarding sharing utilities like that, i wasnt sure where the best level for this would be, what do you think?